kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/requirement/KubernetesRequirement.java

```diff
@@ -34,4 +43,16 @@ public void check(RequiresKubernetes context) throws UnsatisfiedRequirementExcep
                 "Error while checking kubernetes version: [" + e.getMessage() + "]");
         }
     }
+
+    private Config getConfig() {
```

Comment:
I can see same code used in openshift as well except `extension name`. How about moving it to separate class, and use it both the places, to avoid duplication.